src/main/java/wa/client/Client.java

```diff
@@ -1074,6 +1074,8 @@ private void initialize(Properties props) {
         String noSsl = props.getProperty("nossl", "false");
         watsonSsl = noSsl.equalsIgnoreCase("false");
         tenantID = props.getProperty("tenantID");
+        if (tenantID == null)
```

Comment:
Instead of this, in the line above where the property value is retrieved, you can add `, ""`.  Then, if the property doesn't exist (would return null) it will return `""` instead.